d2bs/kolbot/libs/bots/BaalHelper.js

```diff
@@ -342,11 +342,14 @@ MainLoop:
 		Pather.moveTo(15134, 5923);
 		Attack.kill(544); // Baal
 		Pickit.pickItems();
+		if (!Pather.getPortal(null, Config.Leader) || !Pather.usePortal(null, Config.Leader)) {
```

Comment:
If BaalHelper is your last script and there is no leader portal, this change will open a portal go to town and then exit the game, I think the behavior you want is to go to town only if there is a tp otherwise do nothing and wait for next script to start (which will auto go to town).

```
if (Pather.getPortal(null, Config.Leader)) {
    Pather.usePortal(null, Config.Leader);
}
```